Cubical/Displayed/Universe.agda

```diff
@@ -0,0 +1,33 @@
+{-
+
+  - UARel given by a universe and equivalences
+  - SubstRel and DUARel for the element family over the universe
+
+-}
+{-# OPTIONS --cubical --no-import-sorts --no-exact-split --safe #-}
+module Cubical.Displayed.Universe where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Univalence
+
+open import Cubical.Displayed.Base
+open import Cubical.Displayed.Subst
+
+private
+  variable
+    ℓA ℓ≅A ℓB ℓ≅B ℓP : Level
+
+𝒮-Univ : ∀ ℓ → UARel (Type ℓ) ℓ
+𝒮-Univ ℓ .UARel._≅_ = _≃_
+𝒮-Univ ℓ .UARel.ua _ _ =
+  isoToEquiv (invIso (Univalence.isoThm pathToEquiv pathToEquivRefl))
```

Comment:
You can use https://github.com/agda/cubical/blob/master/Cubical/Foundations/Univalence.agda#L207

Or take the inverse equiv of univalence?

Reply:
Ah, I didn't notice `univalenceIso`. I was avoiding inverse equiv to not do unnecessary `isoToEquiv`s, though it probably doesn't matter much here.

Reply:
Right, I suspected that was the reason